### DISCUSSION TITLE
Error message when installing the plugin in OJS-3-3-0-14

### DISCUSSION THREAD

**Discussion by g0007244** [community/community#56143] [Programming Help]
### Select Topic Area

Question

### Body

<div class="cooked"><p>Hello<br>
I have successfully installed <strong>ojs-3.3.0-14</strong> on a subdomain of an online <strong>cPanel</strong> host. (Of course, when I installed my database, I chose <strong>SQL</strong> by default and I don’t know if it is better to choose <strong>SQL</strong> or <strong>SQLI</strong> or etc…)<br>
But when I try to install plugins online from the plugin gallery, I get the following error.</p>
<hr>
<p><strong>The uploaded plugin archive does not contain a folder that corresponds to the plugin name.</strong></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://forum.pkp.sfu.ca/uploads/default/original/3X/3/6/36ed3cda840babf6141e8f051f0e7b9acaaab619.png" data-download-href="https://forum.pkp.sfu.ca/uploads/default/36ed3cda840babf6141e8f051f0e7b9acaaab619" title="5"><img src="https://forum.pkp.sfu.ca/uploads/default/optimized/3X/3/6/36ed3cda840babf6141e8f051f0e7b9acaaab619_2_690x388.png" alt="5" data-base62-sha1="7PU3TBrKClKuiiOGm7ptmpwFfWV" width="690" height="388" srcset="https://forum.pkp.sfu.ca/uploads/default/optimized/3X/3/6/36ed3cda840babf6141e8f051f0e7b9acaaab619_2_690x388.png, https://forum.pkp.sfu.ca/uploads/default/optimized/3X/3/6/36ed3cda840babf6141e8f051f0e7b9acaaab619_2_1035x582.png 1.5x, https://forum.pkp.sfu.ca/uploads/default/original/3X/3/6/36ed3cda840babf6141e8f051f0e7b9acaaab619.png 2x" data-dominant-color="DFE3E5" loading="lazy" style="aspect-ratio: 690 / 388;"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">5</span><span class="informations">1370×772 36.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<hr>
<p>Please tell me step by step what to do? I have no knowledge of PHP, but I can find and replace the codes if I’m properly guided. I have read and checked many messages but I don’t know what exactly should I do?<br>
Thank you very much for your patient and detailed answer, my good friends</p>
<p>In order to better master my own system, I will share the following essential information with you so that I can be guided better.</p>
<p><strong>-------------------------------------------------------------------</strong></p>
<p><strong>My OJS system information is in the link below.</strong></p>
<p><a href="https://ojs.hisiter.ir/index.php/System-Information/index" class="onebox" target="_blank" rel="noopener nofollow ugc">https://ojs.hisiter.ir/index.php/System-Information/index</a></p>
<p><strong>-------------------------------------------------------------------</strong></p>
<p><strong>My Server Information</strong></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://forum.pkp.sfu.ca/uploads/default/original/3X/6/9/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6.png" data-download-href="https://forum.pkp.sfu.ca/uploads/default/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6" title="6"><img src="https://forum.pkp.sfu.ca/uploads/default/optimized/3X/6/9/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6_2_550x500.png" alt="6" data-base62-sha1="f4PNTlOh1xQAFFzhZpJYRxuSdSu" width="550" height="500" srcset="https://forum.pkp.sfu.ca/uploads/default/optimized/3X/6/9/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6_2_550x500.png, https://forum.pkp.sfu.ca/uploads/default/original/3X/6/9/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6.png 1.5x, https://forum.pkp.sfu.ca/uploads/default/original/3X/6/9/69ac8ad8d35c6059afafa3ced95f0b1fed9708b6.png 2x" data-dominant-color="F1EFEF" loading="lazy" style="aspect-ratio: 550 / 500;"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">6</span><span class="informations">692×628 27.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<p><strong>-------------------------------------------------------------------</strong></p>
<p><strong>PageHandler.inc.php</strong> settings in <strong>lib/pkp/controllers/page/PageHandler.inc.php</strong></p>
<p><img src="https://forum.pkp.sfu.ca/uploads/default/original/3X/4/c/4cdd45e3a1ea124006f5ad1f08f5015103e73d8f.png" alt="8" data-base62-sha1="aXYmvikXzFA6hXQTgBTZdSkediL" width="652" height="188" loading="lazy" style="aspect-ratio: 652 / 188;"></p>
<p><strong>-------------------------------------------------------------------</strong></p>
<p><strong>config.inc.php</strong></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://forum.pkp.sfu.ca/uploads/default/original/3X/d/1/d17750484dfe13e06443929c7aacae850c93d44c.png" data-download-href="https://forum.pkp.sfu.ca/uploads/default/d17750484dfe13e06443929c7aacae850c93d44c" title="9"><img src="https://forum.pkp.sfu.ca/uploads/default/original/3X/d/1/d17750484dfe13e06443929c7aacae850c93d44c.png" alt="9" data-base62-sha1="tT1q0I75zx4W4NF6PTgklAkhanq" width="690" height="432" data-dominant-color="F3F1F1" loading="lazy" style="aspect-ratio: 690 / 432;"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">9</span><span class="informations">773×484 14 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p>
<p><strong>-------------------------------------------------------------------</strong></p>
<p>Select <strong>OAI setting</strong> during installation</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://forum.pkp.sfu.ca/uploads/default/original/3X/a/d/adc8b9191bc72b6ba9ac5a11024d468c003d0eb1.png" data-download-href="https://forum.pkp.sfu.ca/uploads/default/adc8b9191bc72b6ba9ac5a11024d468c003d0eb1" title="7"><img src="https://forum.pkp.sfu.ca/uploads/default/original/3X/a/d/adc8b9191bc72b6ba9ac5a11024d468c003d0eb1.png" alt="7" data-base62-sha1="oNmEgJTukIc61758itscDlkdguR" width="690" height="204" data-dominant-color="FAFAFA" loading="lazy" style="aspect-ratio: 690 / 204;"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">7</span><span class="informations">1000×297 12.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><p></p></div>

**Comment by tuves** [comment]
Thanks for posting in the GitHub Community, @g0007244! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
